runtime/doc/news.txt

```diff
@@ -289,6 +289,9 @@ The following new APIs and features were added.
 
 • Terminal buffers respond to OSC background and foreground requests. |default-autocmds|
 
+• |'foldtext'| can be set to an empty string to disable and render the line
+as normal with regular highlighting.
```

Comment:
Please Consider moving up this closer to L148:
> 'foldtext' now supports virtual text format.

Reply:
Looks like it doesn't work properly with would-be wrapped lines in the first place:
![image](https://github.com/neovim/neovim/assets/31730729/ad518843-c7af-4e3f-a075-c104b98eae97)

Reply:
I did notice this at some point but just assumed it was my terminal being funky. And look into this now I know it's related to wrapping.

Reply:
Do you have a reliable reproducer?

Reply:
In a small window: `build/bin/nvim src/nvim/drawline.c +"se foldtext=" +"norm Gkzfj"`

Reply:
Actually with `--clean` the above crashes with the following assert:

    nvim: /home/luuk/dev/neovim/src/nvim/tui/tui.c:1510: tui_raw_line: Assertion `(size_t)attrs[c - startcol] < kv_size(tui->attrs)' failed.

Reply:
I'm not able to reproduce the weird colors but I think I've fixed the tui crash.